core/shared/src/main/scala/scalaz/zio/Fiber.scala

```diff
@@ -24,11 +24,21 @@ package scalaz.zio
 trait Fiber[+E, +A] { self =>
 
   /**
-   * Joins the fiber, with suspends the joining fiber until the result of the
+   * Observes the fiber, which suspends the joining fiber until the result of the
+   * fiber has been determined.
+   */
+  def observe: IO[Nothing, ExitResult[E, A]]
+
+  /**
+   * Joins the fiber, which suspends the joining fiber until the result of the
    * fiber has been determined. Attempting to join a fiber that has been or is
    * killed before producing its result will result in a catchable error.
```

Comment:
This is some old Scaladoc that should be updated. If you join a fiber that has errored, you can catch that error; but you can't catch any error resulting from interruption.